### PR TITLE
fix 'Sign in to the TravisCI via playground doesn't work' (close #651)

### LINKED ISSUE #651: Sign in to the TravisCI via playground doesn't work

### PR DESCRIPTION
**Pull request by miherlosev**
@churkin @inikulin

### CODE REVIEW

File: src/client/sandbox/code-instrumentation/properties/index.js [hunk 1]
```diff
@@ -413,6 +414,26 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 set: (el, value) => el.setAttribute('manifest', value)
             },
 
+            onload: {
+                condition: el => domUtils.isDomElement(el) && domUtils.isImgElement(el),
```
Comment (inikulin) [inline src/client/sandbox/code-instrumentation/properties/index.js]: We need a comment which briefly describes why we need it and points out to the github discussion

**Comment by inikulin** [comment]
In general lgtm, with just minor issue
\r-

**Comment by miherlosev** [comment]
FPR

**Comment by inikulin** [comment]
@miherlosev 

> We need a comment which briefly describes why we need it and **points out to the github discussion**

**Comment by miherlosev** [comment]
FPR

**Comment by miherlosev** [comment]
ping @inikulin @churkin

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Pull request merged**